javascript/javascript2/week2/hyfbayhelper/main.js

```diff
@@ -0,0 +1,189 @@
+const products = getAvailableProducts();
+console.log(products);
+const button = document.getElementById('click');
+const ul1 = document.getElementById('list');
+let filFlag = false;
+let result = [];
+let pageNo = 1;
+function createList(prods) {
+    const uid = document.getElementById('product');
+    uid.innerHTML = '';
+    for (const prod of prods) {
```

Comment:
try writing es6 template string to write HTML.

Reply:
👍🏻 sure,Thank You.